config/settings/production.py

```diff
@@ -12,9 +12,6 @@
 
 ALLOWED_HOSTS = env.str("ALLOWED_HOSTS").split(",")
 
-# Configure Sentry Logging
-SENTRY_DSN = env.str("SENTRY_DSN", "")
-
 if SENTRY_DSN:
     sentry_sdk.init(
```

Comment:
Can you move this code block up as well and the imports?

Reply:
@miltontony I'm not sure I understand this part but please updated.

Reply:
Sorry, It's only meant for the Sentry related configs. So if you look at this "if SENTRY_DSN:" bit and the code below it, as well as the `import sentry_sdk..` statements at the top of this file -- since you're adding the env variable to the `base.py`, my suggestion was that you move these bits of code into `base.py` as well so that they are all in the same file.

Reply:
@miltontony Adding this to base produces the failing tests. I'm not sure if this is permitted?
ERROR: /home/travis/build/praekeltfoundation/higher-health-django/config/settings/base.py Imports are incorrectly sorted.

File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/sentry_sdk/utils.py", line 114, in __init__
    raise BadDsn("Unsupported scheme %r" % parts.scheme)
sentry_sdk.utils.BadDsn: Unsupported scheme ''